README.md

```diff
@@ -1,10 +1,12 @@
 # Scraped
 
-Write declarative scrapers in Ruby
+Write declarative scrapers in Ruby.
+
+If you need to write a webscraper (especially one that hits a page which lists a load of other pages, and jumps into each of _those_ pages to pull out the same data) the `scraped` gem will help you write it quickly and clearly.
```

Comment:
Does that 'especially' go too far in that direction? I find `scraped` really useful even when it's only scraping a single page.